pkg/pillar/Dockerfile

```diff
@@ -45,9 +48,15 @@ RUN [ -z "$GOARCH" ] || export CC=$(echo /*-cross/bin/*-gcc) ;\
     echo "Running go vet" && go vet ./... && \
     echo "Running go fmt" && ERR=$(gofmt -e -l -s $(find . -name \*.go | grep -v /vendor/)) && \
        if [ -n "$ERR" ] ; then echo "go fmt Failed - ERR: "$ERR ; exit 1 ; fi && \
-    make DISTDIR=/out/opt/zededa/bin build
+    make DEV=$DEV DISTDIR=/out/opt/zededa/bin build
 
 WORKDIR /
+
+RUN if [ ${DEV} = "y" ]; then \
+    CGO_ENABLED=0 go get -ldflags "-s -w -extldflags '-static'" github.com/go-delve/delve/cmd/dlv && \
```

Comment:
This is the only thing that concerns me. We are getting the latest `dlv`, whatever it is. That might lead to version mismatches with go (it shouldn't, but it can happen), or pulling in bugs or whatnot.

On the other hand, since this is _only_ in dev-builds, it doesn't matter quite as much.

Can we `go get` a specific version or commit, which is stored in this `Dockerfile`? That would tie it down enough for a dev build. I think eventually, it should be included in `lfedge/eve-alpine` and be installed as part of our usual deploys.